api/swagger-spec/oapi-v1.json

```diff
@@ -13819,6 +13819,11 @@
      "resources": {
       "$ref": "v1.ResourceRequirements",
       "description": "the desired compute resources the build should have"
+     },
+     "maxDuration": {
```

Comment:
then it's a precise technical term that's being used incorrectly.  deadlines are points in time, not lengths of time.  a length of time can be used to compute a deadline given a starting time, but it's not a deadline by itself.

completionDeadlineSeconds it is.